File: fib-gen.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
  Scanner scanner = new Scanner(System.in);
  System.out.println("Fibonacci Sequence Length: ");
  int numberOfIterations = Integer.parseInt(scanner.nextLine());
  int n1 = 1;
  int n2 = 2;
  int iterator = 0;
  while (iterator <= numberOfIterations){
    System.out.println("Fib(" + iterator + ")=" + n1);
    int new_number = n1 + n2;
    n1 = n2;
    n2 = new_number;
    iterator++;
    }
  }
}

File: fib-gen.rs
// Fibonacci sequence generator in Rust
/* Fibonacci sequence generator in Rust */
fn main() {
    let mut n1: i16 = 1;
    let mut n2: i16 = 2;
    let mut iter: i16 = 0;
    let number_of_iterations: i16 = 10;
    
      while iter <= number_of_iterations {
        println!("Fib({})={}",iter,n1);
        let new_number: i16 = n1 + n2;
        n1 = n2;
        n2 = new_number;
        iter += 1;
      }
  }
  
